docker/Dockerfile

```diff
@@ -33,6 +33,10 @@ WORKDIR /app
 
 COPY Gemfile Gemfile.lock ./
 
+# natively compile grpc and protobuf to support alpine musl (dialogflow-docker workflow)
```

Comment:
let probably add the link to the github issue we raised in a comment as well :D

Reply:
sure